backend/pkg/schema/config.go

```diff
@@ -30,6 +33,10 @@ func (c *Config) Validate() error {
 		return nil
 	}
 
+	if c.Timeout <= 5 {
+		c.Timeout = 5
+	}
```

Comment:
If you intend to set a default we should be consistent with the other config structs and introduce a `SetDefaults()` method, that is called by it's parent config struct and then will be executed before Validate() is called.